sass/main.scss

```diff
@@ -726,6 +764,21 @@ footer {
                 width: 100%;
             }
         }
+
+        #hackathon-buttons {
```

Comment:
I've got some code in #1036 that deals with this, so no need for this bit of code
(But thank you! 😄 )